trackpy/static.py

```diff
@@ -0,0 +1,264 @@
+from scipy.spatial import cKDTree
+import random
+import numpy as np
+from warnings import warn
+
+
+def pairCorrelation2D(feat, cutoff, fraction = 1., dr = .5, p_indices = None, ndensity=None, boundary = None,
```

Comment:
pep8: no upper case in function name (make it `pair_correlation_2d` for instance). And remove spaces around =